py/entry_objects.py

```diff
@@ -0,0 +1,75 @@
+class UpcomingEvent:
```

Comment:
if we have Milestones, Exec Updates, and AlumniAdvice as children of Entries, why is an UpcomingEvent not a child?